### PR TITLE
feat(seed): Japanese translation of basic html challenges

### PR DESCRIPTION
**Pull request by t-cool**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [x] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #XXXXX

#### Description

Japanese translation: 01-responsive-web-design/basic-html-and-html5.json

### CODE REVIEW

**Comment by t-cool** [comment]
I sent a new PR!

**Review by raisedadead — CHANGES_REQUESTED** [review]
Hi @t-cool 

1. Remove the lock file from your commit.
2. Amend your commit message to `feat(seed): Japanese translation of basic html challenges`

**Comment by t-cool** [comment]
@raisedadead 
I fixed it!

**Comment by t-cool** [comment]
@raisedadead 
What does "status: blocked" mean?

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+package-lock.json
```
Comment (raisedadead) [inline .gitignore]: Why are you adding the lock file to `.gitignore`? This has to be reverted

**Review by raisedadead — CHANGES_REQUESTED** [review]
Please revisit my https://github.com/freeCodeCamp/freeCodeCamp/pull/17055#pullrequestreview-111878283

You need to remove (DO NOT delete) the package-lock.json from this pull request.

If you need assistance. Please visit the [contributors chat room](https://gitter.im/FreeCodeCamp/Contributors)

**Comment by t-cool** [comment]
I'll close this PR and make another PR tomorrow!

**Pull request closed without merging**